scout/server/blueprints/variants/templates/variants/components.html

```diff
@@ -82,8 +82,20 @@
       <div>
         <strong>Local (arch.)</strong>: {{ variant.local_obs_old or 0 }} obs.
       </div>
+      {% if variant.cosmic_ids %}
+      <strong>CosmicIDs</strong>:
+      <div>
+        {% for cosmicID in variant.cosmic_ids %}
+        cosm{{cosmicID}}
```

Comment:
This might not work with new changes where we store COSMIC as a complete string. See this PR: https://github.com/Clinical-Genomics/scout/pull/1715

New COSMIC IDs might have COSV in them, so it will show the wrong ID here. It might bloat the code by adding conditions to check if cosmicID is int or str. But there might be other way to do more properly?

Reply:
Is this correctly understood?

* Cosmic IDs are stored as either 1. `1234` or 2. `COSV1234`.
* Cosmic IDs should be displayed as 1. `cosm1234` or 2. `COSV1234`

Could be done nicely in the code with a Jinja filter
```
    {% for cosmicID in variant.cosmic_ids %}
        cosmicID | cosmicPrefix


    @app.template_filter()
    def cosmicPrefix(cosmicID):
        if (cosmicID.startswith("COSV"):

       # etc etc


```

Reply:
You're very close.

- COSMIC IDs start with "COS" all the time. they could be COSV, COSM, or COSN (legacy id, but they can still exist)
- If they are `int` type, then add `COSM` otherwise show them as they are.